### PR TITLE
Test feeding junk to commands

### PR DESCRIPTION
**Pull request by t-wissmann**
This adds a test case that tries to call commands with a junk argument.
Of course this should not affect the coverage, but I hope that with
this, we will find bugs in parser exception handling earlier (such as
pr #590).

### CODE REVIEW

**Comment by t-wissmann** [comment]
This should fail until #899 is merged.

File: tests/test_completion.py [hunk 1]
```diff
@@ -231,3 +231,24 @@ def test_posix_escape(hlwm):
...
+@pytest.mark.parametrize("args_before", [0, 1, 2])
+@pytest.mark.parametrize("junk_arg", ['junk', '234', ' '])
+def test_junk_args_dont_crash(hlwm, args_before, junk_arg):
+    commands = hlwm.call('list_commands').stdout.splitlines()
+    for cmd_name in commands:
+        if cmd_name in ['wmexec', 'quit']:
+            continue
+        full_cmd = [cmd_name]
+        for _ in range(0, args_before):
+            # find some arg apropriate for the current command
```
Comment (blarz) [inline tests/test_completion.py:248]: *appropriate

Comment (t-wissmann) [reply tests/test_completion.py]: Thanks! It's fixed.

Comment (blarz) [reply tests/test_completion.py]: Thanks!

**Pull request merged**